packages/passport-integration/test/permissions.spec.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  AppendToFolderAction,
+  AppendToFolderPermission,
+  PCDActionType,
+  PCDCollection,
+  PCDPermissionType
+} from "@pcd/pcd-collection";
+import { PCDPackage } from "@pcd/pcd-types";
+import { SemaphoreIdentityPCDPackage } from "@pcd/semaphore-identity-pcd";
+import { expect } from "chai";
+
+describe("Permissions", async function () {
+  const packages: PCDPackage[] = [SemaphoreIdentityPCDPackage];
+
+  it("executing actions should succeed with right permission", async function () {
```

Comment:
let's break these tests down by action type - here you have only append to, but we also have replace. another case to test for appending is whether the pcd exists there or not already - append would fail whereas replace wouldn't.

Reply:
Done